e2e/tests/home.test.ts

```diff
@@ -9,7 +9,18 @@ test.describe('Home', () => {
     expect(await page.title()).toBe('Home – MOD.UK Design System')
   })
 
-  test('has the correct heading', async ({ page }) => {
-    await expect(page.getByRole('heading')).toHaveText('Updates')
+  test('has the correct level 1 heading', async ({ page }) => {
+    await expect(page.getByRole('heading', { level: 1 })).toHaveText('Design and build consistent Defence services')
+  })
+
+  test('navigates to the get started page after clicking the start button', async ({ page }) => {
+    await page.getByRole('button', { name: 'Get started' }).click()
+
+    await expect(page).toHaveURL(/.*\/get-started\//)
+    await expect(page.getByRole('heading', { level: 1 })).toHaveText('Get started')
+  })
+
+  test('has the correct level 2 heading', async ({ page }) => {
+    await expect(page.getByRole('heading', { level: 2 })).toHaveText('Updates')
```

Comment:
Small opinionated comment I wonder if testing that the button navigates to `/get-started` so we're keeping the test scope limited to the home page.

Reply:
My thinking was that the URL could match what's expected but it could still be a 404 page etc., and the URL isn't important from the user's perspective. So checking one thing on the page (e.g. the first heading) would give more reassurance that it's the right page.

Maybe we could add e.g. a 404 scanner later.

Reply:
Will merge for now and we can have a think when there are more manually added links in.